.github/workflows/ci.yml

```diff
@@ -49,3 +49,18 @@ jobs:
       - uses: actions/checkout@v3
       - name: Run tests
         run: make test-swift
+
+  static-stdlib:
+    strategy:
+      matrix:
+        os: [ubuntu-20.04]
+    runs-on: ${{ matrix.os }}
+    steps:
+      - uses: swift-actions/setup-swift@v1
+        with:
+          swift-version: '5.8.0'
+      - name: Install dependencies
+        run: sudo apt-get install -y libcurl4-openssl-dev
```

Comment:
Ok, the reason why this worked for me locally was that `swift:5.8-focal` comes with the `curl` dev library:

```
$ docker run --rm -it swift:5.8-focal
root@docker-desktop:/host# apt list --installed | grep curl
libcurl3-gnutls/now 7.68.0-1ubuntu2.16 arm64 [installed,local]
libcurl4-openssl-dev/now 7.68.0-1ubuntu2.18 arm64 [installed,local]
libcurl4/now 7.68.0-1ubuntu2.18 arm64 [installed,local]
```

`swift-actions/setup-swift` doesn't seem to be installing this.

I also didn't realise that `static-stdlib` isn't running `make build-for-static-stdlib-docker`. It be possible to do that via docker-in-docker (we do that on Gitlab) but it's probably not worth fiddling with that.

CI passes now: https://github.com/finestructure/swift-dependencies/actions/runs/4686542179